src/main/java/com/google/devtools/build/lib/remote/util/DigestUtil.java

```diff
@@ -43,12 +46,13 @@ public DigestUtil(XattrProvider xattrProvider, DigestHashFunction hashFn) {
     this.digestFunction = getDigestFunctionFromHashFunction(hashFn);
   }
 
+  private static final ImmutableSet<String> DIGEST_FUNCTION_NAMES =
+      Arrays.stream(DigestFunction.Value.values()).map(v -> v.name()).collect(toImmutableSet());
```

Comment:
btw – I could not find a java style guide for the bazel source code, so I assume you use 4 space continuations based on other parts of the codebase

Reply:
What helped me for contributions: The style guide is https://google.github.io/styleguide/javaguide.html and you can use https://github.com/google/google-java-format and its IDE plugins for formatting.

Reply:
Oh of course it's the google Java style guide. Thanks @fmeum!